test/AutoRest.TestServer.Tests/Mgmt/TestProjects/TestProjectTests.cs

```diff
@@ -593,6 +593,8 @@ public IEnumerable<Type> FindAllCollections()
             }
         }
 
+        protected Type FindModelFactory() => MyTypes().Single(IsModelFactory);
```

Comment:
I believe we need to make this nullable and use `SingleOrDefault` here because there are occasions that a project does not have a model factory (for example, for a very small project, and all models inside it happens to meet the condition of omitting in model factory)

Reply:
OK then